adoc/reference-webui-systems.adoc

```diff
@@ -2935,22 +2937,31 @@ The following JSON data is an example of the exported information in the file:
 }
 ----
 
-
 For more information, see the man page on your {productname} server for [command]``virtual-host-gatherer``:
 
 ----
-{prompt.user}man virtual-host-gatherer
+man virtual-host-gatherer
 ----
 
 
-The `README` file coming with the package provides background information about the `type` of a hypervisor, etc.:
+The `README` file provided with the package provides background information about the `type` of a hypervisor, etc.:
 
 ----
 /usr/share/doc/packages/virtual-host-gatherer/README.md
 ----
 
+The man page and the `README` file also contain example configuration files for your reference.
+
+ 
+
 === Configuring Virtual Host Managers via XMLRPC API
 
+{productname} ships with a tool called [command]``virtual-host-gatherer`` that can connect to VHMs using their API, and request information about virtual hosts.
```

Comment:
`virtual-host-gatherer` has nothing to do with the XMLRPC API, and it feels like the wrong place to describe it here.

The XMLRPC API is (not only in this particular case) meant to give functional equivalents to features present in the Web UI, in a machine/programmer friendly way. So in this case what one can do via Web UI is to view, create, edit and delete VHMs - the XMLRPC API offers the same functionality.

Once the VHMs are stored in the database with their configuration, a nightly/configurable/manually triggerable Taskomatic job will use this information as an input to `virtual-host-gatherer`, which will in turn contact the configured VHMs via their API, collect their data and store it in the database.

From the Taskomatic job's POV it is totally irrelevant whether VHM configuration was created/edited via the Web UI, the XMLRPC API or a combination of both. The same is true for `virtual-host-gatherer`: it is once again totally irrelevant whether its input comes from Taskomatic or a manual invocation (see "File-based").

Conceptually, `virtual-host-gatherer` knows nothing about the XMLRPC API or Taskomatic.

I guess a better solution might be to create a small title for this part.

Hope this helps!

Reply:
OK, so I've moved the info around a little and hopefully made it a bit more sane. Let me know what you think.